continuousflex/protocols/protocol_genesis.py

```diff
@@ -164,10 +167,9 @@ def _defineParams(self, form):
                            "to accelerate NM integration, however can make the simulation unstable.",
                        condition="simulationType==2 or simulationType==4",expertLevel=params.LEVEL_ADVANCED)
         group.addParam('nm_mass', params.FloatParam, default=10.0, label='NM mass',
-                      help="Mass value of Normal modes for NMMD", condition="simulationType==2 or simulationType==4",
+                      help="Mass value of Normal modes for NMMD. Lower values accelerate the fitting but can make the "
+                           "simulation unstable", condition="simulationType==2 or simulationType==4",
```

Comment:
We are all guilty, but it is better if we extract constants instead of using numbers of choices in simulationType==2 or simulationType==4

Reply:
You are right, this is an old code I need to update it, in new code I do it